docs/buildah-manifest-add.md

```diff
@@ -0,0 +1,86 @@
+# buildah-manifest-add "1" "September 2019" "buildah"
+
+## NAME
+
+buildah\-manifest\-add - Add an image to a manifest list or image index.
+
+## SYNOPSIS
+
+**buildah manifest add** *listName* *imageName*
```

Comment:
should this be 
```
**buildah manifest add** *listName*|*imageIndex* *imageName*
```

Reply:
Went with `*listNameOrIndexName*` because I think the grouping for `|` might have been confusing.